app/libs/spawnWrapper.js

```diff
@@ -1,15 +1,21 @@
 import childProcess from 'child_process';
-import fixPath from 'fix-path';
+import path from 'path';
 import os from 'os';
 
 const isWindows = process.platform === 'win32';
 
 export default {
   getSpawn: (command, arr) => {
-    fixPath();
-    let path = process.env.PATH;
-    if (isWindows && process.env.PATH.indexOf('Docker') === -1) {
-      path = `C:\\ProgramData\\DockerDesktop\\version-bin;C:\\Program Files\\Docker\\Docker\\Resources\\bin;${process.env.PATH}`;
+    if (
+      isWindows &&
+      process.env.PATH &&
+      process.env.PATH.indexOf('Docker') === -1
```

Comment:
Add parenthesis around statement
```suggestion
      (process.env.PATH.indexOf('Docker') === -1)
```